crates/std_lib/src/ecdsa_secp256k1.nr

```diff
@@ -1,2 +1,2 @@
 #[foreign(ecdsa_secp256k1)]
-fn verify_signature(_signature :[64]u8, _message : []u8, _public_key_x : [32]u8, _public_key_y : [32]u8) -> Field {}
\ No newline at end of file
+fn verify_signature(_public_key_x : [32]u8, _public_key_y : [32]u8, _signature :[64]u8, _message : []u8) -> Field {}
```

Comment:
Why were these re-ordered?

Reply:
They are re-ordered so that this order matches with the order in which they are defined in barretenberg, so that we do not need an additional mapping that will pass them on the correct order to the barretenberg call, depending on the function (in short; we do not need to add specific mapping in the prepare inputs function).